### PR TITLE
Fix/fixed type def for upload stream

### PR DESCRIPTION
**Pull request by patrick-tolosa**
Fixed upload_stream() type definition

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -838,9 +838,9 @@ declare module 'cloudinary' {
 
             function upload_large(path: string, callback?: ErrorCallBack): Promise<any>;
 
-            function upload_stream(upload_preset: string, options?: UploadApiOptions, callback?: ErrorCallBack): Promise<any>;
-
-            function upload_stream(upload_preset: string, callback?: ErrorCallBack): Promise<any>;
+            function upload_stream(options?: UploadApiOptions, callback?: ErrorCallBack): Promise<any>;
```
Comment (asisayag2) [inline types/index.d.ts]: @patrick-tolosa , shouldn't we only add '?' to the upload_preset?

Comment (patrick-tolosa) [reply types/index.d.ts]: It doesn't exist in the function declaration at all, so it doesn't look at like it's optional, it just doesn't exist:

`exports.upload_stream = function upload_stream(callback, options = {})`

Comment (asisayag2) [reply types/index.d.ts]: When uploading an unsigned upload, it's mandatory. 
@strausr how does that work if the method doesn't get an upload_preset?

Comment (patrick-tolosa) [reply types/index.d.ts]: @asisayag2  - There's a separate method for unsigned uploads: 

`exports.unsigned_upload_stream = function unsigned_upload_stream(upload_preset, callback, options = {})`

this one does take the upload_preset argument

**Comment by asisayag2** [comment]
@patrick-tolosa , please make sure Travis passes before merging

**Pull request merged**